### DISCUSSION TITLE
Scenario Improvement: Change all base costs.

### DISCUSSION THREAD

**Discussion by grahambilley** [OpenRCT2/OpenRCT2#15734] [Feature requests]
**OS:** [Windows 10]
**Version:** [0.2.4]
**Commit/Build:** [d645338]

This is an enhancement request. I'm putting this here per request of Aaron van Geffen.

When creating a new scenario, allow the user to specify all base costs. Examples include:

Staff wages: Make handymen and mechanics much more expensive would make scenario play much more difficult.

Landscaping costs, ride build costs, ride run costs, etc.

Credit @James103 for expanding the original idea.

Thank you,

**Comment by James103** [comment]
This improvement could be generalized as "Scenario Improvement: Change all base costs". Generalizing it like that could allow for not just modifying the staff wages, but also various ride, shop, maintenance costs and more. For example, you could make rides cost much more to maintain per dynamic element (lift hill, booster, train, etc) to force players to optimize for least of those dynamic elements, or make landscaping much more expensive as a middle ground between allowing and disallowing it entirely.

**Comment by grahambilley** [comment]
Great idea @James103! I'm going to update the initial comment.
